search-books/src/main/java/nix/education/service/suggestions/solr/SolrSuggestionService.java

```diff
@@ -0,0 +1,11 @@
+package nix.education.service.suggestions.solr;
+
+import java.util.Map;
+import lombok.SneakyThrows;
+import org.apache.solr.client.solrj.SolrClient;
+
+public interface SolrSuggestionService {
```

Comment:
Can we use more generic name for the interface? SuggestionService, for instance? This will enable to reuse this interface for ES impl as well

Reply:
Renamed

Reply:
Changed to SuggestionService